com.unity.multiplayer.mlapi/Prototyping/NetworkAnimator.cs

```diff
@@ -76,7 +76,10 @@ public void NetworkSerialize(NetworkSerializer serializer)
 
         public void SetParamTracking(int paramIndex, bool isTracking)
         {
-            if (paramIndex >= 32) return;
+            if (paramIndex >= 32)
```

Comment:
I doesn't warn on magic numbers ? :-)

Reply:
yeah, bots and tools are not that clever yet :D